scripts/seed_db.py

```diff
@@ -0,0 +1,160 @@
+              " Number Are You At?"
+moving, notMoving = {}, {}
+assignments = ["None | "] * 391
+
+
+class Project:
+    def __init__(self, project_url, attempted_challenges):
+        self.project_url = project_url
+        self.attempted_challenges = attempted_challenges
+        self.table_number = ""
+
+    def __str__(self):
+        return str(self.table_number) + " " + str(self.project_url)
+
+
+# table number to numeric value: A1 -> 1, N15 -> 210
+def table_to_number(table):
+    letter = table[0].upper()
+    num = table[1:]
+    return (ord(letter) - 65) * 15 + int(num)
+
+
+# numeric value to table number: 210 -> N15, 1 -> A1
+def number_to_table(number):
+    letter = chr((int(number) - 1)//15 + 65)
+    num = int(number) - (int(number) - 1)//15 * 15
+    return str(letter) + str(num)
+
+
+# checks if hacker responds with a table number, so they can't move
+def needs_to_stay(response):
+    return re.search('([a-zA-Z]+\d+)', response)
+
+
+# best domain name......
+def format_challenges(attempted_challenges):
+    if attempted_challenges is "":
+        return ""
+    output = []
+    challenges = attempted_challenges.split(',')
+    for challenge in challenges:
+        data = challenge.split(' - ')
+        prize = {
+            'company': data[1],
+            'challenge': data[0],
+            'winner': 'false'
+        }
+        output.append(prize)
+        json_output = json.dumps(output)
+    return json_output
+
+
+def already_in_db():
+    r = requests.get("http://127.0.0.1:5000/api/projects")
+    data = re.findall('\"project_name\": \"(.+)\"', r.text)
+    projects = list(map(lambda x: x.strip(), data))
+    return projects
+
+
+# parses devpost csv and separates hackers into two groups
+# can't move: assigns table and spot, adds notMoving list
+# can move: adds to moving list
+def parse_CSV(reader):
+    already_stored = already_in_db()
+    for row in reader:
+        project_name = row["Submission Title"]
+        project_url = row["Submission Url"]
+        attempted_challenges = format_challenges(row["Desired Prizes"])
+        #attempted_challenges = row["Desired Prizes"]
+        response = row[gdi_devpost]
+
+        name = row['Submission Title'].strip()
+        if name not in already_stored:
+            staying = needs_to_stay(response)
+            if staying is not None:
+                notMoving[project_name] = Project(project_url, attempted_challenges)
+                assignments[table_to_number(staying.group(0))] = name + " | "
+                notMoving[project_name].table_number = staying.group(0)
+            elif "No" not in response and \
+                 "no" not in response and \
+                 "NO" not in response and \
+                 response is not "":
+                print("Manually handle " + project_name)
+                print("Response: " + response)
+            else:
+                moving[project_name] = Project(project_url, attempted_challenges)
+
+
+# fancy way of seeding
```

Comment:
Let's define fancy in the comment and any assumptions that are made when using this one.